### PR TITLE
Update Kube support matrix

### PR DESCRIPTION
**Pull request by youngnick**
Signed-off-by: Nick Young <ynick@vmware.com>

### CODE REVIEW

**Comment by jpeach** [comment]
Are these changes related? What's the problem with the zoom link?

**Comment by youngnick** [comment]
We shouldn't have public Zoom links because of the risk of Zoom bombing. Was just trying to combine two small PRs - they are not otherwise related.

**Comment by youngnick** [comment]
Just the Kube changes.

File: site/_resources/kubernetes.md [hunk 1]
```diff
@@ -11,13 +11,16 @@ The `client-go` package includes a [compatibility matrix][2] as to what Kubernet
...
-| Kubernetes version | Contour v1.0.0 | Contour v1.0.1 | Contour v1.1.0 | Contour v1.2.0 | Contour v1.2.1 |
-| ------------ | :-----------: | :-----------: | :----------: | :--------: |
-| 1.13.x | Supported | Supported | Supported | Not Supported<sup>1</sup> | Not Supported<sup>1</sup> |
-| 1.14.x | Supported | Supported | Supported | Not Supported<sup>1</sup> | Not Supported<sup>1</sup> |
-| 1.15.x | Supported | Supported | Supported | Supported | Supported |
-| 1.16.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Supported |  Supported |
-| 1.17.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Supported | Supported |
+| Contour Version | Kubernetes Version        |
+| --------------- | :------------------- |
+| 1.4.0           | 1.17.x, 1.16.x, 1.15.x   |
```
[the remark this answers is not in the document]
Comment (youngnick) [reply site/_resources/kubernetes.md:16]: 1.18 is not supported in 1.4, only in master.

[the remark this answers is not in the document]
Comment (youngnick) [reply site/_resources/kubernetes.md:16]: I did consider adding a master line tbh.

File: site/_resources/kubernetes.md [hunk 2]
```diff
@@ -11,13 +11,16 @@ The `client-go` package includes a [compatibility matrix][2] as to what Kubernet
...
+| 1.3.0           | 1.17.x, 1.16.x, 1.15.x   |
```
[the remark this answers is not in the document]
Comment (youngnick) [reply site/_resources/kubernetes.md:17]: sure.

**Comment by youngnick** [comment]
I added `master` as an option, I think I'll do this for the Envoy one as well if everyone approves.

**Pull request merged**